pkg/watermark/publish/options.go

```diff
@@ -34,6 +34,10 @@ type publishOptions struct {
 	// Whether it is sink publisher or not
 	// isSource and isSink should not be both true
 	isSink bool
+	// isReduce to indicate it is reduce publisher
+	isReduce bool
```

Comment:
Use `isToVertexReduce` to void confusion?

Reply:
let's delete this option.